packages/mobile/src/fiatExchanges/FiatExchangeOptions.tsx

```diff
@@ -118,26 +116,20 @@ function PaymentMethodRadioItem({
 function FiatExchangeOptions({ route, navigation }: Props) {
   const { t } = useTranslation(Namespaces.fiatExchangeFlow)
   const isAddFunds = route.params?.isAddFunds ?? true
-  const account = useSelector(currentAccountSelector)
   const localCurrency = useSelector(getLocalCurrencyCode)
-  const {
-    SIMPLEX_DISABLED,
-    MOONPAY_DISABLED,
-    KOTANI_SUPPORTED,
-    PONTO_SUPPORTED,
-  } = useCountryFeatures()
+  const { MOONPAY_DISABLED, KOTANI_SUPPORTED, PONTO_SUPPORTED } = useCountryFeatures()
   const pontoEnabled = useSelector(pontoEnabledSelector)
   const kotaniEnabled = useSelector(kotaniEnabledSelector)
   const showPonto = pontoEnabled && PONTO_SUPPORTED
   const showKotani = kotaniEnabled && KOTANI_SUPPORTED
 
   Logger.debug(`Ponto: ${pontoEnabled} Kotani: ${kotaniEnabled}`)
 
+  const isCusdCashInOptionAvailable = !MOONPAY_DISABLED
+  const isCeloCashInOptionAvailable = !MOONPAY_DISABLED
```

Comment:
not sure I understand this correctly, 
cUSD Cash In should be available even if MoonPay is disabled, since you can do it via Simplex.
🤔

Reply:
O.O This would have been a bad bug to have, thanks for catching it!
I originally had Simplex in this condition as well, but then I implemented #6225 which enables Simplex for everyone and removed Simplex from the condition rather than putting the variable as `true`. Fixed!